about/citations/index.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+slug: about
+title: About
+---
+<div class="people-page">
+  <h1>Citing MoveIt</h1>
+  <p>If you use <b>MoveIt</b> in your research, we kindly ask you to include one of the following citations in your publications:
+  <div class="container-fluid">
+    <div class="container">
+      <div class="main-card-wrapper">
+        <div class="main-card-single boarder main-card-single-padding">
+          <img class="mx-auto d-block quote-img" src="/assets/images/people_page/quote.png">
+          <p>David Coleman, Ioan A. Șucan, Sachin Chitta, Nikolaus Correll,
+          Reducing the Barrier to Entry of Complex Robotic Software: a MoveIt! Case Study,
+          <em>Journal of Software Engineering for Robotics</em>,
+          5(1):3–16, May 2014. doi: <a href="http://dx.doi.org/10.6092/JOSER_2014_05_01_p3">10.6092/JOSER_2014_05_01_p3</a>.</p>
+          <p class="text-center"><a class="button" href="https://arxiv.org/pdf/1404.3785">PDF</a>
+          <a class="button" href="http://dx.doi.org/10.6092/JOSER_2014_05_01_p3">publisher</a></p>
```

Comment:
please capitalize all these buttons
```suggestion
          <a class="button" href="http://dx.doi.org/10.6092/JOSER_2014_05_01_p3">Publisher</a></p>
```

Reply:
Done. Last commit capitalizes labels of buttons.